### PR TITLE
Fixes a game breaking roboticist bug

### PR DESCRIPTION
**Pull request by jbox144**
## About The Pull Request

Fixes a major, game breaking roboticist bug. This bug has made roboticist practically unplayable due to how bad it is. Like, seriously immensly bad I couldn't believe it had been unnoticed for this long.

Roboticsts now get their suitskirt roundstart as per prefs. Previously, regardless of preferences you wouldn't get a suitskirt roundstart as a roboticist, but now you will.

Also fixes a bug which made the "coordination calibration" internal damage unfixable for mechs, making it now fixable as expected. This bug caused me a lot of embarrassing defeats, so I'm very happy to see it fixed.

## Why It's Good For The Game

My immersion is ruined having to get a new suitskirt every shift.

Also, having a very dangerous type of internal damage unfixable due to a bug is a bit lame. It's caused a lot of embarrassing defeats. This is technically a salt PR!

## Changelog
:cl:
fix: Fixed a bug which made the "coordination failure" mecha internal damage unfixable. Technically this is a salt PR...
fix: Fixed a bug which meant the roboticist didn't get a suitskirt roundstart even if suitskirt prefs are enabled.
/:cl:

### CODE REVIEW

File: austation/code/modules/clothing/under/jobs/medisci.dm [hunk 1]
```diff
@@ -46,6 +46,18 @@
...
+	desc = "It's a slimming black with reinforced seams; great for industrial work."
+	icon = 'austation/icons/obj/clothing/uniforms.dmi'
+	alternate_worn_icon = 'austation/icons/mob/uniform.dmi'
+	icon_state = "robotics_skirt"
+	item_state = "robotics"
+	item_color = "robotics_skirt"
+	body_parts_covered = CHEST|GROIN|ARMS
+	can_adjust = FALSE
+	fitted = FEMALE_UNIFORM_TOP
+
```
Comment (TheFakeElon) [inline austation/code/modules/clothing/under/jobs/medisci.dm:60]: this goes in engineering.dm, not medsci. Yeah I know it's weird but that's how normal robo is done

all you have to do is remove the /engineering, the "rnd" is not needed

original code:
`/obj/item/clothing/under/rank/roboticist`

**Review by TheFakeElon — CHANGES_REQUESTED** [review]

File: austation/code/modules/clothing/under/jobs/medisci.dm (hunk 1 above)
Comment (jbox144) [reply austation/code/modules/clothing/under/jobs/medisci.dm:60]: it wouldn't work like that as the skirt varient has to be a subclass of the original varient, and the original varient is written as
`/obj/item/clothing/under/rank/rnd/roboticist/skirt`

Comment (TheFakeElon) [reply austation/code/modules/clothing/under/jobs/medisci.dm:60]: I am aware, I was quoting the original robo jumpsuit code. Look at the original code for the skirt variant in engineering.dm. This code would work too but I'm just trying to keep things consistent :)

Comment (park66665) [reply austation/code/modules/clothing/under/jobs/medisci.dm:60]: Well, to clear up, there are two roboticist's jumpsuit definitions: `/obj/item/clothing/under/rank/rnd/roboticist` (rnd.dm) and `/obj/item/clothing/under/rank/roboticist` (engineering.dm). Def without `rnd` in it is considered *obsolete*, so making a new jumpskirt def with `rnd` actually is the way; and can you remove obsolete definitions to clarify things?

Comment (TheFakeElon) [reply austation/code/modules/clothing/under/jobs/medisci.dm:60]: Oh weird. Yeah just comment out the engineering.dm one then :P

Comment (park66665) [reply austation/code/modules/clothing/under/jobs/medisci.dm:60]: *SS13 shitcode strikes again*

File: austation/code/modules/vending/wardrobes.dm [hunk 2]
```diff
@@ -11,7 +11,7 @@
 	aus_products = list(/obj/item/clothing/under/rank/cargo/tech/skirt = 3)
 
 /obj/machinery/vending/wardrobe/robo_wardrobe
-	aus_products = list(/obj/item/clothing/under/rank/engineering/roboticist/skirt = 2)
+	aus_products = list(/obj/item/clothing/under/rank/rnd/roboticist/skirt = 2)
```
Comment (MCterra10) [inline austation/code/modules/vending/wardrobes.dm:14]: amazing, quite possibly the best crack I ever smoked

**Review by MCterra10 — APPROVED** [review]

File: austation/code/modules/vending/wardrobes.dm (hunk 2 above)
Comment (jbox144) [reply austation/code/modules/vending/wardrobes.dm:14]: y e s

**Pull request merged**